e2e/tests/contracts.rs

```diff
@@ -1476,6 +1478,7 @@ async fn test_contract_submit_and_response() -> Result<()> {
     let contract_methods = contract_instance.methods();
 
     let submitted_tx = contract_methods.get(1, 2).submit().await?;
+    tokio::time::sleep(Duration::from_millis(500)).await;
```

Comment:
What's with these sleep statements over here?

Reply:
Flakiness when running the tests against the new core binary. If you ask for the transaction status too soon after you submitted it, the core will complain about the transaction not existing.